cluster/kubectl.sh

```diff
@@ -32,15 +32,17 @@ set -o pipefail
 
 KUBE_ROOT=${KUBE_ROOT:-$(dirname "${BASH_SOURCE[0]}")/..}
 source "${KUBE_ROOT}/cluster/kube-util.sh"
-source "${KUBE_ROOT}/cluster/clientbin.sh"
+source "${KUBE_ROOT}/hack/lib/util.sh"
 
 # If KUBECTL_PATH isn't set, gather up the list of likely places and use ls
 # to find the latest one.
 if [[ -z "${KUBECTL_PATH:-}" ]]; then
-  kubectl=$( get_bin "kubectl" "cmd/kubectl" )
+  kubectl=$( kube::util::find-binary "kubectl" )
```

Comment:
did you check that these have the same search paths etc?

Reply:
the logic looks the same.
`::find-binary`:
https://github.com/kubernetes/kubernetes/blob/master/hack/lib/util.sh#L147-L221
`clientbin.sh`:
https://github.com/kubernetes/kubernetes/blob/master/cluster/clientbin.sh#L23-L96

Reply:
ack